internal/deviations/deviations.go

```diff
@@ -676,4 +683,6 @@ var (
 	_ = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
 
 	_ = flag.Bool("deviation_storage_component_unsupported", false, "Set to true for device that does not support telemetry path /components/component/storage")
+
+	_ = flag.Bool("deviation_skip_controller_card_power_admin", false, "Set to true if power-admin-state config on controller card should be skipped.")
```

Comment:
Don't add new flags for deviations, use the metadata exclusively:
https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md

Reply:
Addressed. Thanks.